### PR TITLE
ZCS-12081 remove zimlets as part of upgrade to zimbra10

### PR DESCRIPTION
**Pull request by RakeshAMore**
ZCS-12081 remove zimlets as part of an upgrade to zimbra10

Remove the following zimlets:

1. zimbra-zimlet-briefcase-edit-lool
2. zimbra-zimlet-restore-contacts
3. zimbra-zimlet-duplicate-contacts

### CODE REVIEW

File: rpmconf/Install/Util/utilfunc.sh [hunk 1]
```diff
@@ -1715,8 +1715,8 @@ removeExistingPackages() {
   fi
 }
 
-removeZextrasPackagesIfInstalled() {
```
Comment (deepakzimb) [inline rpmconf/Install/Util/utilfunc.sh:1718]: can we keep it as before and create new method to remove deprecated zimlets

**Comment by umagmrit** [comment]
@RakeshAMore we need to undeploy these  zimlets. For reference please see https://github.com/Zimbra/zm-build/pull/220/files#diff-fb24c3fa9332e8033d4ae6d656d4bc0c593ae5851b947c5ee9808455fe70aea4R6586

File: rpmconf/Install/Util/utilfunc.sh (hunk 1 above)
Comment (RakeshAMore) [reply rpmconf/Install/Util/utilfunc.sh:1718]: I thought it to keep a common method for removing not required packages.

Comment (RakeshAMore) [reply rpmconf/Install/Util/utilfunc.sh]: Done

**Review by deepakzimb — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]

**Pull request merged**